src/scheduler/scheduler.py

```diff
@@ -0,0 +1,48 @@
+from datetime import date
+
+import schedule
+
+from database.database import (
+    get_broadcast_subscribers,
+    get_raw_menu,
+    get_hidden_cuisines
+)
+from scheduler.scheduler_config import BREAKFAST_BROADCAST_TIME, DINNER_BROADCAST_TIME
+from util.const import BREAKFAST, DINNER
+from util.messages import menu_msg
+from util.util import parse_menu
+
+
+def scheduler(bot):
+    breakfast_subscribers = get_broadcast_subscribers(BREAKFAST)
+    dinner_subscribers = get_broadcast_subscribers(DINNER)
+
+    # schedule breakfast and dinner broadcasts
+    schedule.every().day\
```

Comment:
Just take note that scheduler uses busy waiting, which is more resource-intensive. If the VM is dying for idk what reason, might want to convert this to cron instead.